src/blt_mad/helpers.go

```diff
@@ -20,7 +16,8 @@ func RemoveZeros(data []float64) ([]float64, error) {
 	}
 
```

Comment:
earlier in this file, `nonZeros` is counted by testing equality of a float (`if value != 0.0 {`) which is generally bad practice. And one way of handling this is more correctly is to test against a low threshold (e.g., `if math.Abs(value) < 0.00001`)

BUT: it made me wonder, why are these arrays of frequencies being passed around as floats? Are they not just integer counts?